lib/Runtime/Debug/TTRuntimeInfoTracker.cpp

```diff
@@ -774,6 +774,7 @@ namespace TTD
 
         this->EnqueueRootPathObject(_u("global"), ctx->GetGlobalObject());
         this->EnqueueRootPathObject(_u("null"), ctx->GetLibrary()->GetNull());
+        this->EnqueueRootPathObject(_u("undeclBlockVar"), Js::RecyclableObject::FromVar(ctx->GetLibrary()->GetUndeclBlockVar()));
```

Comment:
>undeclBlockVar [](start = 40, length = 14)

Can you give some context on this change?

Reply:
Right, sorry -- for some reason, my changes caused TTD tests to break. @mrkmarron found that the issue was that somehow, this undeclBlockVar was ending up in the heap now in ways it wasn't before -- @boingoing has a better explanation there. The problem with that is that its typeID is TypeIds_Null, which the TTD code assumed meant the global `null` alone before. This change allows the TTD code to track undeclBlockVar separately from the normal `null` type, so that replaying doesn't hit an assert.